### ISSUE TITLE
Images are being overwritten without the -force option

### ISSUE THREAD

**Issue by jdtournier** [jdtournier/mrtrix3#169]
I can overwrite images without the `-force` option:

```ShellSession
$ mrconvert orig.nii poo.nii
mrconvert: [100%] copying from "orig.nii" to "poo.nii"...
$ mrconvert orig.nii poo.nii
mrconvert: [100%] copying from "orig.nii" to "poo.nii"...
```

This is not good. We need to check through the history and figure out what happened...

**Comment by maxpietsch** [comment]
Not sure whether that helps: -force does not work with type_image_out(). It seems to work with type_file_out() though.

**Comment by jdtournier** [comment]
Looks like that was my mistake - introduced when I made a few changes to allow a 'confirm overwrite' dialog box on the GUI. Hopefully the above commit will fix it...

**Comment by jdtournier** [comment]
closed by commit 20d02ab
